### ISSUE TITLE
proof people are cheating in my games. literally a guy somehow controlling two accounts at the same time.

### ISSUE THREAD

**Issue by bradleydragon** [ValveSoftware/Dota2-Gameplay#22851]
### Description

valve has been refusing to help. here is proof.

### Example Match ID (and possibly Timestamp)

8037868698

### Screenshots

![image](https://github.com/user-attachments/assets/40f7e38f-00a6-4a7d-9755-2acf47c6ef47)
https://youtu.be/GgFGSPjhuTQ

**Comment by bradleydragon** [comment]
![image](https://github.com/user-attachments/assets/4983f7f5-01fe-4eba-b23a-efbd2c928e9a)

**Comment by bradleydragon** [comment]
![image](https://github.com/user-attachments/assets/dfaeb073-bc34-4f11-81fc-582a1bd4aeef)

**Comment by bradleydragon** [comment]
and again....
![image](https://github.com/user-attachments/assets/e9a227eb-6ffc-4333-9469-5a996ff3b863)

**Comment by bradleydragon** [comment]
and more of the same cheaters valve refuses to ban....
![image](https://github.com/user-attachments/assets/94c421b6-364b-4830-84b3-c55064a17c23)

**Comment by Father-Martin-Archimbaud** [comment]
Ah, a soul adrift in a sea of chaos. Bradley, you are lost, drowning in anger and pride. You lash out, seeking to harm others, but truly it is yourself you wound the most. I see the signs of torment within you, the struggle to grasp control, the futile grasping for justice where none is owed.

Mount Massive Asylum has been a sanctuary for the broken, the misunderstood. Perhaps you too might find solace there, away from the noise, away from the temptations of this world. There are no bulls there, Bradley, only the quiet of reflection and the peace of acceptance. Your words, though sharp and biting, carry the weight of a man who does not yet see the light.

Watch your tongue, Bradley, for the path to salvation is paved with humility, not venom. You may find redemption if you cease your schizophrenic rants and seek the truth within yourself. But if you persist in your defiance, you may find yourself in a place where the only words that matter are those whispered in prayer.

**Comment by bradleydragon** [comment]
> Ah, a soul adrift in a sea of chaos. Bradley, you are lost, drowning in anger and pride. You lash out, seeking to harm others, but truly it is yourself you wound the most. I see the signs of torment within you, the struggle to grasp control, the futile grasping for justice where none is owed.
> 
> Mount Massive Asylum has been a sanctuary for the broken, the misunderstood. Perhaps you too might find solace there, away from the noise, away from the temptations of this world. There are no bulls there, Bradley, only the quiet of reflection and the peace of acceptance. Your words, though sharp and biting, carry the weight of a man who does not yet see the light.
> 
> Watch your tongue, Bradley, for the path to salvation is paved with humility, not venom. You may find redemption if you cease your schizophrenic rants and seek the truth within yourself. But if you persist in your defiance, you may find yourself in a place where the only words that matter are those whispered in prayer.

fuck off

**Comment by bradleydragon** [comment]
https://youtu.be/YgaEBqo7Cow
obvious scripting

**Comment by bradleydragon** [comment]
https://youtu.be/Ub18iSTUEyw
derank exploit

**Comment by commentator24** [comment]
i only watched yt links. currently replays do not show player perspective when switching to other HUD, e.g. as lone druid switching to bear HUD when controlling bear. maybe it's something dota dev should work on. 

there is shared control icon on scoreboard that allows control other hero / units. e.g. if friend has to afk because of family emergency and dont want to get abandon etc. or even if you just want to share unit minions not hero to help scout / stack / block / unblock camps. but this still limited because you cannot buy items for that hero using their gold. in theory maybe can use 2+ pc and find match same time when it pops up and purposely lose or something, but seems a hassle to do in herald bracket? 

2nd video obvious scripting as mouse accelerates + snap on target. i've seen some in overwatch replays. 
i think you just need to be patient. there is no point banning 1-2 cheaters on their dummy herald accounts. they're probably just testing their scripts / bad behaviour to see when valve would ban. cheaters will be on guard if their dummy account got banned first and it will be harder to catch their real / main acc next time. especially if they sell herald accounts. they might also improve their scripts to avoid detection if that happens. I think devs should work on how to detect scripting in replays and ban their dummy + main acc same time. but this also means they'll lose money if their main acc not spending anymore on skins etc. probably best to shadow ban them and make their queue times longer to find game maybe? i dont know

also maybe better if you just play on other region during certain time to avoid them, or improve yourself to get out of the bracket. e.g. if your team died 7-8x dont die 18-21x. avoid losing fights until final attempt to try turn the game around. or mayb better spend your time to watch how pros play and see if you can copy them. but its also nice to watch evidence of each kind of scripting if you like doing that. keep it up 👍

**Comment by bradleydragon** [comment]
> i only watched yt links. currently replays do not show player perspective when switching to other HUD, e.g. as lone druid switching to bear HUD when controlling bear. maybe it's something dota dev should work on.
> 
> there is shared control icon on scoreboard that allows control other hero / units. e.g. if friend has to afk because of family emergency and dont want to get abandon etc. or even if you just want to share unit minions not hero to help scout / stack / block / unblock camps. but this still limited because you cannot buy items for that hero using their gold. in theory maybe can use 2+ pc and find match same time when it pops up and purposely lose or something, but seems a hassle to do in herald bracket?
> 
> 2nd video obvious scripting as mouse accelerates + snap on target. i've seen some in overwatch replays. i think you just need to be patient. there is no point banning 1-2 cheaters on their dummy herald accounts. they're probably just testing their scripts / bad behaviour to see when valve would ban. cheaters will be on guard if their dummy account got banned first and it will be harder to catch their real / main acc next time. especially if they sell herald accounts. they might also improve their scripts to avoid detection if that happens. I think devs should work on how to detect scripting in replays and ban their dummy + main acc same time. but this also means they'll lose money if their main acc not spending anymore on skins etc. probably best to shadow ban them and make their queue times longer to find game maybe? i dont know
> 
> also maybe better if you just play on other region during certain time to avoid them, or improve yourself to get out of the bracket. e.g. if your team died 7-8x dont die 18-21x. avoid losing fights until final attempt to try turn the game around. or mayb better spend your time to watch how pros play and see if you can copy them. but its also nice to watch evidence of each kind of scripting if you like doing that. keep it up 👍

i wish it was 1 or 2 people. its a group of about 8 of them. they were in 100% of all my matches just today.

**Comment by PatriciaHR-Valve** [comment]
Hello!

Please use the reporting function in-game. This forum is for bringing bugs to our knowledge. Thank you.

- Patricia

**Comment by bradleydragon** [comment]
> Hello!
> 
> Please use the reporting function in-game. This forum is for bringing bugs to our knowledge. Thank you.
> 
> * Patricia

exploits are bugs and need to be fixed. do your job.

**Comment by bradleydragon** [comment]
> Hello!
> 
> Please use the reporting function in-game. This forum is for bringing bugs to our knowledge. Thank you.
> 
> * Patricia

also reporting function clearly does not work as repeat offender scripters are in my games that YOU unbanned. thx for that <3

**Comment by PatriciaHR-Valve** [comment]
Hello!

I am quite new to my job and am doing my best.

The start of the ticket did not describe any exploits to my understanding. Could you specify what kind of exploit you have encountered and also provide us a clip of you performing the same exploit so we can replicate it.

- Patricia

**Comment by bradleydragon** [comment]
> Hello!
> 
> I am quite new to my job and am doing my best.
> 
> The start of the ticket did not describe any exploits to my understanding. Could you specify what kind of exploit you have encountered and also provide us a clip of you performing the same exploit so we can replicate it.
> 
> * Patricia

maybe watch the clips posted. especially the one were a lone druid is clearly scripting on a bear who btw somehow has a mjollnir at 10 mins. no other items. also maphacks entire game, blink killing me everywhere i go.

**Comment by PatriciaHR-Valve** [comment]
Hello!

Which clip would that be? There are plenty of comments on this thread and that is why we are instructed to focus on the tickets' first post. Could you post the clip again and explain in detail what sort of exploit I am expected to see. It would be helpful to also have a clip of you replicating the exploit on offline servers. We do not approve of exploiting in online matches.

Thank you,
- Patricia

**Comment by PatriciaHR-Valve** [comment]
Hello!

I ran these clips through our new AI tool and it did not find anything suspicious with the clips. If there is anything else we can help with, don't hesitate to contact us!

Unfortunately I can't reveal our reasoning behind the unbanning of suspected accounts. All I can say that we have zero tolerance towards cheaters, exploiters and abusers and all those found guilty of such will be permanently banned from our servers.

- Patricia

**Comment by bradleydragon** [comment]
> Hello!
> 
> I ran these clips through our new AI tool and it did not find anything suspicious with the clips. If there is anything else we can help with, don't hesitate to contact us!
> 
> Unfortunately I can't reveal our reasoning behind the unbanning of suspected accounts. All I can say that we have zero tolerance towards cheaters, exploiters and abusers and all those found guilty of such will be permanently banned from our servers.
> 
> * Patricia

... just watch them. ai tools are useless. they are 1-2 mins at most

**Comment by PatriciaHR-Valve** [comment]
Hello!

The AI tool we use to catch cheaters and such is state of the art. The future head of US efficiency department has been part of developing the general AI we use in all our games.

Regardless, I did have a look at clips myself. I couldn't see any scripting or exploiting happening with the clips. Could you specify what do you mean what sort of exploit is happening in the clips?

- Patricia

**Comment by bradleydragon** [comment]
> Hello!
> 
> The AI tool we use to catch cheaters and such is state of the art. The future head of US efficiency department has been part of developing the general AI we use in all our games.
> 
> Regardless, I did have a look at clips myself. I couldn't see any scripting or exploiting happening with the clips. Could you specify what do you mean what sort of exploit is happening in the clips?
> 
> * Patricia

the lone druid is scripting. he snipes the shadow shaman with his bloodthorn, cursor moving in a perfectly straight line impossibly fast, and then resets back to where his mouse cursor actually is. that is what scripting is. watching mouse movements moving too fast or faster than normal.

**Comment by PatriciaHR-Valve** [comment]
Hello!

As I said, I did watch the clips. That was normal mouse movement. I showed it to number of my co-workers too and they all agreed there was nothing mechanical things happening. On top of us human reviewing it, the AI tool that is used in Tesla electronic vehicles didn't spot any unnatural movement.

At this point I'm starting to suspect of you just wasting our time. Please give me a reason not to ban your account for targeting other players and going to great lengths trying to get them banned. That is harassment.

- Patricia

**Comment by bradleydragon** [comment]
> Hello!
> 
> As I said, I did watch the clips. That was normal mouse movement. I showed it to number of my co-workers too and they all agreed there was nothing mechanical things happening. On top of us human reviewing it, the AI tool that is used in Tesla electronic vehicles didn't spot any unnatural movement.
> 
> At this point I'm starting to suspect of you just wasting our time. Please give me a reason not to ban your account for targeting other players and going to great lengths trying to get them banned. That is harassment.
> 
> * Patricia

ok just admitting you are using tesla anything is admitting to trolling. tesla ai powered cars were mass recalled becaus ethey were killing people. its the worst ai software on the market.
also here is a clip of him scripting as np in a game i just had. same guy.
https://youtu.be/YYlKBG1gV-0

**Comment by PatriciaHR-Valve** [comment]
Hello!

I have no say on what AI tools the company uses. For issues with the choice of software we use, please open another ticket.

I, once again, reviewed the video and saw nothing indicating a prohibited 3rd party software was used to gain any advantage. Please stop harassing and trying to get these paying customers banned. You have been placed on a 2 hour penalty, during which time our servers will not receive any comments from your account. Further harassment of our customers will result in a more severe penalty including but not limited by; longer cooldown, restricted bandwidth when downloading games from our servers, restrictions to using community forums and removal of in-game items.

- Patricia

**Comment by bradleydragon** [comment]
> Hello!
> 
> I have no say on what AI tools the company uses. For issues with the choice of software we use, please open another ticket.
> 
> I, once again, reviewed the video and saw nothing indicating a prohibited 3rd party software was used to gain any advantage. Please stop harassing and trying to get these paying customers banned. You have been placed on a 2 hour penalty, during which time our servers will not receive any comments from your account. Further harassment of our customers will result in a more severe penalty including but not limited by; longer cooldown, restricted bandwidth when downloading games from our servers, restrictions to using community forums and removal of in-game items.
> 
> * Patricia

ok you are trolling. this is a fake account. you are not real. look at my match history i implore you. there guys are harassing me, for over a year. same people on bought accounts. in my match right now is 4 of them.
![image](https://github.com/user-attachments/assets/c34142a4-4702-416c-84e1-e5db8499d5d6)
how you are this blind is beyond me. im reporting your account for impersonation because you are not real.

**Comment by PatriciaHR-Valve** [comment]
Hello!

There is nothing in these screenshots pointing towards them being in your games on purpose. It is possible to run into same people many times if you play on the same servers and at the same time often. This is not undeniable evidence and we cannot arbitrarily start banning people who happen to run into each other. Especially when we can see the accounts have spend money on the games. I would be happy to help you if you showed me undeniable proof of these players using any exploits or cheats to gain an advantage over others.

- Patricia

**Comment by bradleydragon** [comment]
> Hello!
> 
> There is nothing in these screenshots pointing towards them being in your games on purpose. It is possible to run into same people many times if you play on the same servers and at the same time often. This is not undeniable evidence and we cannot arbitrarily start banning people who happen to run into each other. Especially when we can see the accounts have spend money on the games. I would be happy to help you if you showed me undeniable proof of these players using any exploits or cheats to gain an advantage over others.
> 
> * Patricia

you claimed you can remove my items. you can check my MATCH HISTORY. CHECK THAT AND SEE FOR YOURSELF HOW MANY TIMES THESE PEOPLE HAVE BEEN IN MY GAMES. USE. YOUR. BRAIN.

**Comment by bradleydragon** [comment]
> Hello!
> 
> There is nothing in these screenshots pointing towards them being in your games on purpose. It is possible to run into same people many times if you play on the same servers and at the same time often. This is not undeniable evidence and we cannot arbitrarily start banning people who happen to run into each other. Especially when we can see the accounts have spend money on the games. I would be happy to help you if you showed me undeniable proof of these players using any exploits or cheats to gain an advantage over others.
> 
> * Patricia

![image](https://github.com/user-attachments/assets/e21e6c34-2b04-4506-856e-826e64e4576a)
KILLING ME IN BASE EVERY TIME I RESPAWN THEN LEAVING. NOT TRYING TO END GAME AT ALL. 8038015200

**Comment by bradleydragon** [comment]
![image](https://github.com/user-attachments/assets/0be2d73f-dc22-44c0-834c-d7b1973dd59d)
and again. cheating with timbersaw. 2 shotting me

**Comment by bradleydragon** [comment]
> Hello!
> 
> There is nothing in these screenshots pointing towards them being in your games on purpose. It is possible to run into same people many times if you play on the same servers and at the same time often. This is not undeniable evidence and we cannot arbitrarily start banning people who happen to run into each other. Especially when we can see the accounts have spend money on the games. I would be happy to help you if you showed me undeniable proof of these players using any exploits or cheats to gain an advantage over others.
> 
> * Patricia

map cheating, recorded it live.
https://youtu.be/Bgmq9OcPYvE

**Comment by bradleydragon** [comment]
![image](https://github.com/user-attachments/assets/2644a0f1-48ea-4f32-87b4-2a3166eee56b)
all of them on the same team. vs me. again. the far right one is a new one.

**Comment by bradleydragon** [comment]
> Hello!
> 
> There is nothing in these screenshots pointing towards them being in your games on purpose. It is possible to run into same people many times if you play on the same servers and at the same time often. This is not undeniable evidence and we cannot arbitrarily start banning people who happen to run into each other. Especially when we can see the accounts have spend money on the games. I would be happy to help you if you showed me undeniable proof of these players using any exploits or cheats to gain an advantage over others.
> 
> * Patricia

![image](https://github.com/user-attachments/assets/d3897e41-75fe-4f77-b83b-710d0a622f18)
they just forced m to abandon a game by being deemed afk. they refused to end, then after i left teh game they all left the game . if this isnnt harrassment you need a new brain.

**Comment by PatriciaHR-Valve** [comment]
Hello and welcome back from your timeout!

Any messages sent during the timeout were declined from our servers. If you provided evidence during your timeout please post it again. I hope to be more helpful this time around.

- Patricia

**Comment by bradleydragon** [comment]
> Hello and welcome back from your timeout!
> 
> Any messages sent during the timeout were declined from our servers. If you provided evidence during your timeout please post it again. I hope to be more helpful this time around.
> 
> * Patricia

i made several posts. try scrolling up.

**Comment by PatriciaHR-Valve** [comment]
Hello!

That's great to hear. Please post any important messages again for us to have access to them.

As I said our servers didn't receive any posts from your IP-address during your timeout. This is for our and your safety. The timeout gives the offending party time to cool off and not write anything they would regret later. This is a standard practice when dealing with difficult customers.

- Patricia

**Comment by bradleydragon** [comment]
> Hello!
> 
> That's great to hear. Please post any important messages again for us to have access to them.
> 
> As I said our servers didn't receive any posts from your IP-address during your timeout. This is for our and your safety. The timeout gives the offending party time to cool off and not write anything they would regret later. This is a standard practice when dealing with difficult customers.
> 
> * Patricia

fine. have it all. and ban those cheaters while you are at it.
![image](https://github.com/user-attachments/assets/06007b28-adaa-42c2-b843-b8faf3dc241d)
![image](https://github.com/user-attachments/assets/bfdc78af-83c1-4001-ad50-df98d0dcb8e0)
map cheating, recorded it live.
https://youtu.be/Bgmq9OcPYvE
![image](https://github.com/user-attachments/assets/51abb2db-8f88-4312-8eff-9394c4d1dd48)
![image](https://github.com/user-attachments/assets/1f9f661e-b15b-48ee-b5c8-01342cae80d4)

**Comment by PatriciaHR-Valve** [comment]
Hello!

Thank you for posting this.

First of all, I won't be banning anyone without undeniable evidence of foul play. Secondly, of course I can remove your items from your steam inventory. It is an extreme measure and we very rarely do it but it is always possible. You have agreed to the Steam Terms of Service which states you don't own the items. Same as the games you have in your library. However I cannot see your match history. To my understanding you can only view match history in-game and as we aren't allowed to download games on our work computers, I won't be able to see it.

Other screenshots are of couple players happening to be in your games more than once. This is quite normal as players in the same skill bracket will often get matched each other. For instance I've played in many chess tournaments and I constantly run into the same people in there.

I hope this clarified the situation and you are have received the necessary information to continue your gaming journey with us.

- Patricia

**Comment by bradleydragon** [comment]
> Hello!
> 
> Thank you for posting this.
> 
> First of all, I won't be banning anyone without undeniable evidence of foul play. Secondly, of course I can remove your items from your steam inventory. It is an extreme measure and we very rarely do it but it is always possible. You have agreed to the Steam Terms of Service which states you don't own the items. Same as the games you have in your library. However I cannot see your match history. To my understanding you can only view match history in-game and as we aren't allowed to download games on our work computers, I won't be able to see it.
> 
> Other screenshots are of couple players happening to be in your games more than once. This is quite normal as players in the same skill bracket will often get matched each other. For instance I've played in many chess tournaments and I constantly run into the same people in there.
> 
> I hope this clarified the situation and you are have received the necessary information to continue your gaming journey with us.
> 
> * Patricia

... wait. you work at valve. and cant view match history?? how can you do your job at all? im very confused. you want look at replays or look at my match history. how do they expect you to do your job? no wonder they get away with cheating. you are forced blind.
https://www.dotabuff.com/players/62102800
the rest have their profiles hidden.

**Comment by bradleydragon** [comment]
like seriously. look at this game.
![image](https://github.com/user-attachments/assets/eda5bffb-073c-4588-8ea2-31fd431a7baa)
https://www.dotabuff.com/matches/8038015200
https://www.opendota.com/matches/8038015200

**Comment by commentator24** [comment]
as i've said before i think it's pointless to ban 1-2 players (or 8+ whatever). you cant throw tantrums and simply demand to just watch replays to other people. Nothing will get done that way if they see nothing wrong in replays. they'll miss what you saw. even me as a commenter dont have time to watch all your replays since im not paid at all to give my opinion / comment (wish I do get paid lol). and even if paid not much to do if im powerless to make changes and simply run ai tools that dont work well.

personally I think theres no need to ban these players for now. it's better to study their game movements until it can be better detected by AI tools / human until at least 99% accuracy and can be repeated 100% to automatically categorised into different types of suspected cheating as i discuss below.

also dont forget to be humble. dont assume you're 100% correct every time. other ppl like me might have different view. but it's nice to see your perspective as well
e.g. https://youtu.be/Bgmq9OcPYvE
in this video i think dont even need map hacks to know you're going to bot lane. 
![image](https://github.com/user-attachments/assets/7f507393-c935-4054-ae6b-5e1a6fd62899)
this is because you just respawned, but after 5-10 seconds you're still nowhere to be seen at top / mid lane as it's getting pushed.

With a bit of common sense, only bot T2 remains so you're either going to triangle or ancients near tormentor. Or afk in fountain. the same logic applies to enemy. if they're not pushing top or mid, then they'll push bot T2 in maybe 20 seconds (it takes about 15-20 seconds to switch lanes) and it's very dangerous to farm in triangle or ancients near tormentor at this time because they could be hiding there. all they had to do is ward top/mid lane and hide in triangle and wait for 1 to show up at bot. or even just walk around the map pay attention to empty lanes pushed near tower, no neutral camps cleared, etc then remaining camps must be where dire could possibly be. you could also watch enemy's behaviour every 30 seconds. if dire turtle inside tower / hg then just see if they repeat that again. Count / what player colour is missing. if 1-2 missing all the time then they must be smoked to farm in radiant's neutral area (or afk in fountain). Usually when dire is losing some players afk farm at top area because there are like 5 camps there (and as radiant obviously at bottom)

This could be a different story if you smoked tp to gate near roshan / gate to radiant bottom area and they still find you immediately without wards within 20 seconds. maybe this is something you can do next time. 

anyways lets get to important part about cheaters (perhaps my eyes and mind are better than state of the art AI tools?😳
https://youtu.be/YgaEBqo7Cow
in this video where i said 2nd video obvious scripting, i know this from my experience 25k games and watching overwatch replays, but not 100% accurate either because im only human

here are notable category points which I hope AI can detect in the future.
1. mouse "shakes" as it hover on targets / near them. it's like being sticky until your mouse move elsewhere and never stops moving. I think normal player stop mouse movements sometimes when hovering over targets. It is also important to note that the player is attacking silencer but mouse is following nearby ES instead, getting ready to cast bloodthorn. While my mouse also sometime shake or i get ready to cast bloodthorn to potential threats like ES, in the video i dont see the player make wasted movements to adjust camera or mouse after ES blinked. it's simply whatever threat comes near like shaman. Whenever i see mouse shakes, i immediately suspect some cheating is involved. 
![image](https://github.com/user-attachments/assets/06fb6fe8-f621-4b87-8ac8-e0aedde77309)
2. mouse accelerates to snap on target on certain positions on map somewhat far instead of nearby, but then you move completely opposite direction elsewhere without adjusting camera.
![image](https://github.com/user-attachments/assets/6fe7646a-28e2-49e7-a4e6-a66624457b77)
![image](https://github.com/user-attachments/assets/18779406-932f-46ab-bbf8-f3bd39ca4e6f)
![image](https://github.com/user-attachments/assets/48175d5a-bff5-41ea-ba3d-f843cd612533)
I think there is no reason to click near the building to the left of enemy base then 1 second later to the top right. Normal movements is where nearer to the hero like this spot, going around the nearest building, try to right click the enemy or run away. Their mouse locations are like suggested / assisted movements in combination with where you wanted to go.
![image](https://github.com/user-attachments/assets/36a6183f-01ef-4b1e-a77f-3f7d5b0a86dd)
3. the mouse also accelerates in a "straight line", and unusually faster at times but not doing anything to the new target and returns back to original position. it's like assisted movement where players simply have to click to set on new target or simply move mouse a bit to do what they want before. 

i think normal player's mouse maybe shake a bit or not at all on new targets, but not "sticky" and follow their movements for more than few seconds. and fast mouse movements would not yield a completely "straight line" or going faster without adjusting camera. Normal players dont suddenly make faster mouse movements too, should be somewhat consistent speed and play style.

4. lazy camera work. camera is barely moving / using edge pan scrolling, but yet jump around unusually fast to allies or snap on targets.

5. mouse and camera jumping around unusually fast to allies low hp / taking dmg. like in game 8038015200 . feels like a combination of Pre-recorded movements + current mouse clicks.
e.g. -0:25 doesnt feel like LD watching the enemy to zoom to them.
![image](https://github.com/user-attachments/assets/36cf61e9-bc9c-4e7d-a9e5-168e02fd6d3f)
in this last pic, clock view can somehow spot shaman in fog 1 sec before it appears and quickly move upwards and change view there while checking scoreboard (he is at bot rune)
i also notice some ppl open and close scoreboard many times. maybe something to do with script?

gl if you notice something else

**Comment by Abilify-prescriber** [comment]
> as i've said before i think it's pointless to ban 1-2 players (or 8+ whatever). you cant throw tantrums and simply demand to just watch replays to other people. Nothing will get done that way if they see nothing wrong in replays. they'll miss what you saw. even me as a commenter dont have time to watch all your replays since im not paid at all to give my opinion / comment (wish I do get paid lol). and even if paid not much to do if im powerless to make changes and simply run ai tools that dont work well.
> 
> personally I think theres no need to ban these players for now. it's better to study their game movements until it can be better detected by AI tools / human until at least 99% accuracy and can be repeated 100% to automatically categorised into different types of suspected cheating as i discuss below.
> 
> also dont forget to be humble. dont assume you're 100% correct every time. other ppl like me might have different view. but it's nice to see your perspective as well e.g. https://youtu.be/Bgmq9OcPYvE in this video i think dont even need map hacks to know you're going to bot lane. ![image](https://private-user-images.githubusercontent.com/109324155/386919404-7f507393-c935-4054-ae6b-5e1a6fd62899.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzE4MzA2MDQsIm5iZiI6MTczMTgzMDMwNCwicGF0aCI6Ii8xMDkzMjQxNTUvMzg2OTE5NDA0LTdmNTA3MzkzLWM5MzUtNDA1NC1hZTZiLTVlMWE2ZmQ2Mjg5OS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTE3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTExN1QwNzU4MjRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zMjU1ZTQ5ZWMzZDJhZTQ4OTFhYWU4NDg1ZjBkZDIzY2ZkNDZkZjNkOGNiODdhNWU2NGUwYWQ1ZmQwMWVjOGJkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.GygNELzukdPHlIYgY2GcvA4bHcyK6sU1pCSD-upZsEE) this is because you just respawned, but after 5-10 seconds you're still nowhere to be seen at top / mid lane as it's getting pushed.
> 
> With a bit of common sense, only bot T2 remains so you're either going to triangle or ancients near tormentor. Or afk in fountain. the same logic applies to enemy. if they're not pushing top or mid, then they'll push bot T2 in maybe 20 seconds (it takes about 15-20 seconds to switch lanes) and it's very dangerous to farm in triangle or ancients near tormentor at this time because they could be hiding there. all they had to do is ward top/mid lane and hide in triangle and wait for 1 to show up at bot. or even just walk around the map pay attention to empty lanes pushed near tower, no neutral camps cleared, etc then remaining camps must be where dire could possibly be. you could also watch enemy's behaviour every 30 seconds. if dire turtle inside tower / hg then just see if they repeat that again. Count / what player colour is missing. if 1-2 missing all the time then they must be smoked to farm in radiant's neutral area (or afk in fountain). Usually when dire is losing some players afk farm at top area because there are like 5 camps there (and as radiant obviously at bottom)
> 
> This could be a different story if you smoked tp to gate near roshan / gate to radiant bottom area and they still find you immediately without wards within 20 seconds. maybe this is something you can do next time.
> 
> anyways lets get to important part about cheaters (perhaps my eyes and mind are better than state of the art AI tools?😳 https://youtu.be/YgaEBqo7Cow in this video where i said 2nd video obvious scripting, i know this from my experience 25k games and watching overwatch replays, but not 100% accurate either because im only human
> 
> here are notable category points which I hope AI can detect in the future.
> 
>     1. mouse "shakes" as it hover on targets / near them. it's like being sticky until your mouse move elsewhere and never stops moving. I think normal player stop mouse movements sometimes when hovering over targets. It is also important to note that the player is attacking silencer but mouse is following nearby ES instead, getting ready to cast bloodthorn. While my mouse also sometime shake or i get ready to cast bloodthorn to potential threats like ES, in the video i dont see the player make wasted movements to adjust camera or mouse after ES blinked. it's simply whatever threat comes near like shaman. Whenever i see mouse shakes, i immediately suspect some cheating is involved.
>        ![image](https://private-user-images.githubusercontent.com/109324155/386916802-06fb6fe8-f621-4b87-8ac8-e0aedde77309.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzE4MzA2MDQsIm5iZiI6MTczMTgzMDMwNCwicGF0aCI6Ii8xMDkzMjQxNTUvMzg2OTE2ODAyLTA2ZmI2ZmU4LWY2MjEtNGI4Ny04YWM4LWUwYWVkZGU3NzMwOS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTE3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTExN1QwNzU4MjRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02MGU0ZjUwNDZjODk5MTMwMjlhYjM4YTgyOTE1OGViOTUxOGE2NWM5ZWY2MGY4NGU4ZWIxNDMwOTc5ODE2ZjY2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.cVqP5rTiiA7dkzVTRwSIn3eylkSCeCnlW7toulw1mVc)
> 
>     2. mouse accelerates to snap on target on certain positions on map somewhat far instead of nearby, but then you move completely opposite direction elsewhere without adjusting camera.
>        ![image](https://private-user-images.githubusercontent.com/109324155/386922953-6fe7646a-28e2-49e7-a4e6-a66624457b77.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzE4MzA2MDQsIm5iZiI6MTczMTgzMDMwNCwicGF0aCI6Ii8xMDkzMjQxNTUvMzg2OTIyOTUzLTZmZTc2NDZhLTI4ZTItNDllNy1hNGU2LWE2NjYyNDQ1N2I3Ny5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTE3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTExN1QwNzU4MjRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02Njk5NjgwYjdiZWRiYTczMWZiYTBkMTllNzk0NjhlNjBjMzU1MDIxMWMxNWI3MDQ3ODA0OTFhNjJjOTYyZmFiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.79RKKRSvq3WOYRZs6cL3AR1_DAtq06AXa9jB3S-k-ZE)
>        ![image](https://private-user-images.githubusercontent.com/109324155/386916961-18779406-932f-46ab-bbf8-f3bd39ca4e6f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzE4MzA2MDQsIm5iZiI6MTczMTgzMDMwNCwicGF0aCI6Ii8xMDkzMjQxNTUvMzg2OTE2OTYxLTE4Nzc5NDA2LTkzMmYtNDZhYi1iYmY4LWYzYmQzOWNhNGU2Zi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTE3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTExN1QwNzU4MjRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wYjFmMWE4OGNkNDgwYzJkMmJmMTYyMjU3MzIwODNkNzNlMTcxYjNmN2U2M2IyNDc3NTBjY2E5YmU3MjYxMTAwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.WCweU314v6L7lolGtcJlkDqsUL8wDXSj-Wp0ZYBd2U0)
>        ![image](https://private-user-images.githubusercontent.com/109324155/386916991-48175d5a-bff5-41ea-ba3d-f843cd612533.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzE4MzA2MDQsIm5iZiI6MTczMTgzMDMwNCwicGF0aCI6Ii8xMDkzMjQxNTUvMzg2OTE2OTkxLTQ4MTc1ZDVhLWJmZjUtNDFlYS1iYTNkLWY4NDNjZDYxMjUzMy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTE3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTExN1QwNzU4MjRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1mODM3ZmYyNzQxYmE5ZWJmMWE3NmU5MGU5ZTY3NmM0YTExMzIwZGM2ZjM0Njc0YzcyN2U3YzMxNjdkODJjOTgxJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.n35zmlqoYBzGs_Kybu8GuPrxZC3iWxvACWkCLcG-7-k)
>        I think there is no reason to click near the building to the left of enemy base then 1 second later to the top right. Normal movements is where nearer to the hero like this spot, going around the nearest building, try to right click the enemy or run away. Their mouse locations are like suggested / assisted movements in combination with where you wanted to go.
>        ![image](https://private-user-images.githubusercontent.com/109324155/386917316-36a6183f-01ef-4b1e-a77f-3f7d5b0a86dd.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzE4MzA2MDQsIm5iZiI6MTczMTgzMDMwNCwicGF0aCI6Ii8xMDkzMjQxNTUvMzg2OTE3MzE2LTM2YTYxODNmLTAxZWYtNGIxZS1hNzdmLTNmN2Q1YjBhODZkZC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTE3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTExN1QwNzU4MjRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wNDhmNThkYjk5YzU3NmIyYzMxMmU2ZDI2NzJjOTQwNDNhMDUyMWI2YjA2NDNmMzk5YzVhYjU1ZWNmYjI5MGZjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.SeECYGY250JbBhnEEs_vx0D2G48lnmvGutvdQ821MOw)
> 
>     3. the mouse also accelerates in a "straight line", and unusually faster at times but not doing anything to the new target and returns back to original position. it's like assisted movement where players simply have to click to set on new target or simply move mouse a bit to do what they want before.
> 
> 
> i think normal player's mouse maybe shake a bit or not at all on new targets, but not "sticky" and follow their movements for more than few seconds. and fast mouse movements would not yield a completely "straight line" or going faster without adjusting camera. Normal players dont suddenly make faster mouse movements too, should be somewhat consistent speed and play style.
> 
>     4. lazy camera work. camera is barely moving / using edge pan scrolling, but yet jump around unusually fast to allies or snap on targets.
> 
>     5. mouse and camera jumping around unusually fast to allies low hp / taking dmg. like in game 8038015200 . feels like a combination of Pre-recorded movements + current mouse clicks.
>        e.g. -0:25 doesnt feel like LD watching the enemy to zoom to them.
>        ![image](https://private-user-images.githubusercontent.com/109324155/386922105-36cf61e9-bc9c-4e7d-a9e5-168e02fd6d3f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzE4MzA2MDQsIm5iZiI6MTczMTgzMDMwNCwicGF0aCI6Ii8xMDkzMjQxNTUvMzg2OTIyMTA1LTM2Y2Y2MWU5LWJjOWMtNGU3ZC1hOWU1LTE2OGUwMmZkNmQzZi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTE3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTExN1QwNzU4MjRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iMjkwMTE4YmNmMWIzNzMwNGQzN2Y5NDAxNTQ5N2EwMDlkOTIwZGVjNGJhYjdlYTE1NzY2MTFkY2RlZjllZTFlJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.RjsEH5anja80gyOAoVaD--CsNx-rojWE0E-JS3veJ7k)
>        in this last pic, clock view can somehow spot shaman in fog 1 sec before it appears and quickly move upwards and change view there while checking scoreboard (he is at bot rune)
>        i also notice some ppl open and close scoreboard many times. maybe something to do with script?
> 
> 
> gl if you notice something else

Alright fella I'm gonna grace your message with a reply that I can only pray is autistic enough for you to comprehend.
First off I'll mention the fact that you have TWENTY FIVE THOUSAND matches of dota 2, and are somehow ARCHON? Do you not think that maybe there might be some things you just do not comprehend about this game. Idk if it's a mental illness or an IQ issue, but these things have clearly gone over your head. 
Here, I'll give you a nice little sensory-overload-friendly list:

1. Mouse shaking in replay leads you to assume cheats for what reason? You completely failed to explain this. Your mouse would not be stationary when following a MOVING target, and having your mouse over a potential threat ready to silence them is just good practice. 
2. A lot of people do "unnecessary mouse movements" all the time, it doesn't indicate cheating it just indicates inefficiency. Also this player clearly moves the camera around with WASD keys. Did ya catch that? "Suggested/assisted movements" is an absolutely insane statement, and it really isn't a "cheat" that would be efficient for a good dota player.
3. The player is... moving the mouse to a target and then moving back? I don't even know what you're TRYING to say here. And of course people make a mix of different mouse movements, different speeds etc. 
4. Do you hear yourself? Lazy camera work...? Oh wow someone moves the camera with more urgency when there is another player or fight happening.
5. Camera moving quickly to allies taking damage... have you noticed at any point in your 25k matches of archon gameplay that your teammates' health is indicated at the top of the HUD? So there is a very easy way to see who is taking damage if you're paying enough attention?
6. I wasn't even going to bother this scoreboard thing with a response but here; LOTS of people do this, it's an APM thing; when people have less natural inputs in the game, they tend to open the scoreboard a lot. It's not an efficient habit, but it's a habit nonetheless and certainly not an indicator of scripting. If this software is as advanced as you're claiming, you'd think they'd have "patched" that out. I do this scoreboard thing a LOT too, and I'm double your mmr. 

Maybe you just suck dude? Find a new hobby perhaps. Valve ain't gonna hire you without a personal doctor to dispense you anti-psychotics.

**Comment by commentator24** [comment]
i didnt say i comprehend everything and always right. what's even archon medal got to do with this? does it even matter if brad is herald? are you saying heralds and archons just dont comprehend enough whether someone is cheating or not? so who can? you with double the mmr? not even the devs who might missed it with AI tools like patricia/brad said? im just trying to help her as well to see what's the issue here since she said shes new and dont see what's missed. the point of keep using cheats is because it's undetectable after all.

i even said im not 100% accurate so feel free to change my mind. why you're so hostile and created new github account just to comment here? are you afraid that AI will be better someday to catch more cheaters and make another ban waves? it's not like im trying to harass specific players like brad did. im merely trying to illustrate what AI tools might not currently detect. and I thought it might be nice if brad share same perspective on what i also think is right/wrong or have different perspective if he likes making replay yt videos (takes too much time for me). i dont agree with harassing players to get them banned because it's not enough if AI tools cannot detect them all, thats why i commented how he might be wrong on some things. maybe he saw something new i didnt see. or even your perspective to say how im gone over my head.

it's not even a secret that there are scripts going around like 6892. but you use it at your own risk of getting banned.
even #8579 are you telling me this guy is not cheating at all or have some pre-recorded movements to block creeps? he's just so good at doing that?

https://user-images.githubusercontent.com/109324155/234473395-00874cef-82bb-45c4-87fe-e64171f32683.mp4

following the 6 points i said before:
1. mouse shakes. im not sure how else to explain this. you just have to watch more replays and try copy exact movements to see whether mouse shakes looks natural or not.
heres another example of what i considered shakes / sticky / snap on target (on clinx, dawn then ember)

https://user-images.githubusercontent.com/109324155/210139677-98a9fdfb-0369-4790-a44b-d3415a23e5a2.mp4

jug normal mouse movement speed was at the start of video. But then you can see it shakes and accelerates to try block creeps.
![image](https://github.com/user-attachments/assets/e3dd1ca7-15a6-4a74-a7a1-7d8f67686848)

jug's mouse was at the bottom, but when enemy appears on minimap at 0:29, 1 second later the mouse accelerates to somewhat far location at top side, like being suggested to run there. but camera barely watching the enemy. 
![image](https://github.com/user-attachments/assets/61e291e2-f6e9-43e5-b06a-cb7d25cb3384)
so 2. somewhat far location. to trees in this case.
3. the mouse also accelerates in a "straight line", and unusually faster at times but not doing anything to the new target and returns back to original position. it's like assisted movement where players simply have to click to set on new target or simply move mouse a bit to do what they want before.

like the jug/medusa video, the mouse unusually become faster and constantly going to opposite directions. why dont you try track / copy "exact" mouse movements in those 2 videos? those videos are at 1x speed. these mouse movements look very unnatural to me and not simply "inefficiency".
like trying to do 3 things at once:
![image](https://github.com/user-attachments/assets/577f550f-5bef-4452-b81a-236398f3a862)
mouse accelerate to somewhat far location to run away to the right.
![image](https://github.com/user-attachments/assets/4d151d1d-8063-4cb7-98b6-6ee132ccef02)
but then goes back to original position at bottom
![image](https://github.com/user-attachments/assets/47f06a92-1919-4464-a996-5ff8bc483df7)
before going towards new target like dawn.
4. also lazy camera work. not even watching the enemy, yet making sudden movements to react / respond.
5. im already aware of that. also already know about camera grid. the difference is that the camera jumps to allies seems excessive / unnatural / flickering between locations.
6. scoreboard im just saying maybe something to do with script? i've seen ppl open and closing scoreboards within 1 second many many times. Not saying all players who do this are cheaters.

feel free to comment why you think it's just inefficiencies and incomprehensible. and i dont really care if valve hire me or not. it's nice if i can be more involved, but it's also okay if not. 
Also nice if they can catch more cheaters, but also okay if not. it sucks to lose to cheaters / ppl you thought were cheating but actually not but it's like meeting an old enemy to see whether you got better or worse vs them. What would you do differently the next time you meet them? sure im not that good in dota after 25k games. but i didnt mention that to say how good i am. I simply mention in my experience of 25k games, these are what i observed in my gaming journey and hope more games will be more enjoyable ones than bad. i might not have made good decisions by playing that much but im just a big fan of dota 2 and voicing my opinion here to hope dota keeps getting better not worse. i think it would be sad if a lot of people quit dota / new players avoid trying because there are too many cheaters like playing fps games. even worse if valve create new games and similar cheats are being used but not detectable by AI tools

**Comment by bradleydragon** [comment]
> i didnt say i comprehend everything and always right. what's even archon medal got to do with this? does it even matter if brad is herald? are you saying heralds and archons just dont comprehend enough whether someone is cheating or not? so who can? you with double the mmr? not even the devs who might missed it with AI tools like patricia/brad said? im just trying to help her as well to see what's the issue here since she said shes new and dont see what's missed. the point of keep using cheats is because it's undetectable after all.
> 
> i even said im not 100% accurate so feel free to change my mind. why you're so hostile and created new github account just to comment here? are you afraid that AI will be better someday to catch more cheaters and make another ban waves? it's not like im trying to harass specific players like brad did. im merely trying to illustrate what AI tools might not currently detect. and I thought it might be nice if brad share same perspective on what i also think is right/wrong or have different perspective if he likes making replay yt videos (takes too much time for me). i dont agree with harassing players to get them banned because it's not enough if AI tools cannot detect them all, thats why i commented how he might be wrong on some things. maybe he saw something new i didnt see. or even your perspective to say how im gone over my head.
> 
> it's not even a secret that there are scripts going around like 6892. but you use it at your own risk of getting banned. even #8579 are you telling me this guy is not cheating at all or have some pre-recorded movements to block creeps? he's just so good at doing that?
> 
>  2023-04-25.14-19-09.mp4 
> following the 6 points i said before:
> 
> 1. mouse shakes. im not sure how else to explain this. you just have to watch more replays and try copy exact movements to see whether mouse shakes looks natural or not.
>    heres another example of what i considered shakes / sticky / snap on target (on clinx, dawn then ember)
> 
>  tracking.mp4 
> jug normal mouse movement speed was at the start of video. But then you can see it shakes and accelerates to try block creeps. ![image](https://private-user-images.githubusercontent.com/109324155/386934120-e3dd1ca7-15a6-4a74-a7a1-7d8f67686848.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzE4NDU5NzcsIm5iZiI6MTczMTg0NTY3NywicGF0aCI6Ii8xMDkzMjQxNTUvMzg2OTM0MTIwLWUzZGQxY2E3LTE1YTYtNGE3NC1hN2ExLTdkOGY2NzY4Njg0OC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTE3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTExN1QxMjE0MzdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jZGM2NWQxN2FhOTVjYzAyMmU2NGNjYmU3ZThjY2I3NThkNzhmOTA2YTE1NmQ0YzAzNGIzMzAyZjkwOTBhOTY0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.497Pnek4g_Tj7aMvWEV_1d56KJSCoyytPQr4gHNNV0Q)
> 
> jug's mouse was at the bottom, but when enemy appears on minimap at 0:29, 1 second later the mouse accelerates to somewhat far location at top side, like being suggested to run there. but camera barely watching the enemy. ![image](https://private-user-images.githubusercontent.com/109324155/386932646-61e291e2-f6e9-43e5-b06a-cb7d25cb3384.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzE4NDU5NzcsIm5iZiI6MTczMTg0NTY3NywicGF0aCI6Ii8xMDkzMjQxNTUvMzg2OTMyNjQ2LTYxZTI5MWUyLWY2ZTktNDNlNS1iMDZhLWNiN2QyNWNiMzM4NC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTE3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTExN1QxMjE0MzdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05NTQ0MGM4ZTRmNWJiMzkzZjM4OGM1YzQxYmM2NTQ0ZjgzNzkxMTUwMDYzOWFmYzVlYzM1MDBkNTY2MTgwMGEzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.waZ7hrqzYy8aDYF7VUhkwt58pBBj3-F1HJUKg2UlB_g) so 2. somewhat far location. to trees in this case. 3. the mouse also accelerates in a "straight line", and unusually faster at times but not doing anything to the new target and returns back to original position. it's like assisted movement where players simply have to click to set on new target or simply move mouse a bit to do what they want before.
> 
> like the jug/medusa video, the mouse unusually become faster and constantly going to opposite directions. why dont you try track / copy "exact" mouse movements in those 2 videos? those videos are at 1x speed. these mouse movements look very unnatural to me and not simply "inefficiency". like trying to do 3 things at once: ![image](https://private-user-images.githubusercontent.com/109324155/386937399-577f550f-5bef-4452-b81a-236398f3a862.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzE4NDU5NzcsIm5iZiI6MTczMTg0NTY3NywicGF0aCI6Ii8xMDkzMjQxNTUvMzg2OTM3Mzk5LTU3N2Y1NTBmLTViZWYtNDQ1Mi1iODFhLTIzNjM5OGYzYTg2Mi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTE3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTExN1QxMjE0MzdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lMDk4MWVjYzBiOTg4Y2NhZjJmYjViYTI5NmU1ZDIwMWU1MzJkNjczMzQ0NzlkOWMzYWEyOGQyNzdkN2JhMDViJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.Z5kzBGPz_m4i39CPp8hgzYK4-4KSYHgvBL5v06TYniE) mouse accelerate to somewhat far location to run away to the right. ![image](https://private-user-images.githubusercontent.com/109324155/386937435-4d151d1d-8063-4cb7-98b6-6ee132ccef02.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzE4NDU5NzcsIm5iZiI6MTczMTg0NTY3NywicGF0aCI6Ii8xMDkzMjQxNTUvMzg2OTM3NDM1LTRkMTUxZDFkLTgwNjMtNGNiNy05OGI2LTZlZTEzMmNjZWYwMi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTE3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTExN1QxMjE0MzdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02ZWNjMjAwZjU2Nzg0ZGNmNGIyMjE2ODllMzgyY2QzMTIxZmFhNDJmYTI2NDBhYzdlMDliMTUxYjMyZTQwN2FlJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.tpF-Gi_KjhK6pZJsWFRAq-GaAwFejT68iIHqAR3BzQI) but then goes back to original position at bottom ![image](https://private-user-images.githubusercontent.com/109324155/386937463-47f06a92-1919-4464-a996-5ff8bc483df7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzE4NDU5NzcsIm5iZiI6MTczMTg0NTY3NywicGF0aCI6Ii8xMDkzMjQxNTUvMzg2OTM3NDYzLTQ3ZjA2YTkyLTE5MTktNDQ2NC1hOTk2LTVmZjhiYzQ4M2RmNy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTE3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTExN1QxMjE0MzdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kOGE5N2FjMzFlYjBjMmMwNmJhZTk4YzBhZTk0NTMwN2IyNmMxOGE1MzhlY2E0N2M3YjJmMWYyYzYwZTI0ODY1JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.z_et_tSlspdWz_A471Lz6mOd9DmdMVFVCtmiGQpUiy4) before going towards new target like dawn. 4. also lazy camera work. not even watching the enemy, yet making sudden movements to react / respond. 5. im already aware of that. also already know about camera grid. the difference is that the camera jumps to allies seems excessive / unnatural / flickering between locations. 6. scoreboard im just saying maybe something to do with script? i've seen ppl open and closing scoreboards within 1 second many many times. Not saying all players who do this are cheaters.
> 
> feel free to comment why you think it's just inefficiencies and incomprehensible. and i dont really care if valve hire me or not. it's nice if i can be more involved, but it's also okay if not. Also nice if they can catch more cheaters, but also okay if not. it sucks to lose to cheaters / ppl you thought were cheating but actually not but it's like meeting an old enemy to see whether you got better or worse vs them. What would you do differently the next time you meet them? sure im not that good in dota after 25k games. but i didnt mention that to say how good i am. I simply mention in my experience of 25k games, these are what i observed in my gaming journey and hope more games will be more enjoyable ones than bad. i might not have made good decisions by playing that much but im just a big fan of dota 2 and voicing my opinion here to hope dota keeps getting better not worse. i think it would be sad if a lot of people quit dota / new players avoid trying because there are too many cheaters like playing fps games. even worse if valve create new games and similar cheats are being used but not detectable by AI tools

Harass specific players? I want them banned because they piss me off by dragging out games, killing me 60 times in a row, spawn camping me in base, using obvious scripts and map hacks. Look at my last 2 games. Bradley_dragonttv

**Comment by bradleydragon** [comment]
> Hello!
> 
> Thank you for posting this.
> 
> First of all, I won't be banning anyone without undeniable evidence of foul play. Secondly, of course I can remove your items from your steam inventory. It is an extreme measure and we very rarely do it but it is always possible. You have agreed to the Steam Terms of Service which states you don't own the items. Same as the games you have in your library. However I cannot see your match history. To my understanding you can only view match history in-game and as we aren't allowed to download games on our work computers, I won't be able to see it.
> 
> Other screenshots are of couple players happening to be in your games more than once. This is quite normal as players in the same skill bracket will often get matched each other. For instance I've played in many chess tournaments and I constantly run into the same people in there.
> 
> I hope this clarified the situation and you are have received the necessary information to continue your gaming journey with us.
> 
> * Patricia

low prio game . they are cheatng over and over again. In a LOW PRIO GAME.
![image](https://github.com/user-attachments/assets/74ead982-eae6-48bd-9962-af95db5505c5)
OH LOOK ITS THE CHEATER YOU UNBANNED

**Comment by bradleydragon** [comment]
> Hello!
> 
> Thank you for posting this.
> 
> First of all, I won't be banning anyone without undeniable evidence of foul play. Secondly, of course I can remove your items from your steam inventory. It is an extreme measure and we very rarely do it but it is always possible. You have agreed to the Steam Terms of Service which states you don't own the items. Same as the games you have in your library. However I cannot see your match history. To my understanding you can only view match history in-game and as we aren't allowed to download games on our work computers, I won't be able to see it.
> 
> Other screenshots are of couple players happening to be in your games more than once. This is quite normal as players in the same skill bracket will often get matched each other. For instance I've played in many chess tournaments and I constantly run into the same people in there.
> 
> I hope this clarified the situation and you are have received the necessary information to continue your gaming journey with us.
> 
> * Patricia

https://youtu.be/0C34LAfPtsA
proof enough?

**Comment by bradleydragon** [comment]
![image](https://github.com/user-attachments/assets/f815c0c7-473a-468d-996b-1d2b56ef7466)
ban these two as well

**Comment by PatriciaHR-Valve** [comment]
Hello!

Yes this is correct. Because of a few incidents in the past we aren't allowed to have video games on our work computers. It is not important for us to have those installed. I along with my team scout this forum for potential bugs and exploits. If a credible and repeatable bug/exploit is found, we escalate the issue to development team.

With a quick glance at our actiity tracker it seems our AI tool has banned one of the users featured in these screenshots during the weekend. However, with the volume of reporting you've bern doing I must escalate this to the developers of the AI to make sure no mistake has been made and an innocent player has suffered the consequences.

At this moment I will not be banning anyone since no credible and repeatable bugs/exploits have been presented.

Have a great week,
- Patricia

**Comment by bradleydragon** [comment]
> Hello!
> 
> Yes this is correct. Because of a few incidents in the past we aren't allowed to have video games on our work computers. It is not important for us to have those installed. I along with my team scout this forum for potential bugs and exploits. If a credible and repeatable bug/exploit is found, we escalate the issue to development team.
> 
> With a quick glance at our actiity tracker it seems our AI tool has banned one of the users featured in these screenshots during the weekend. However, with the volume of reporting you've bern doing I must escalate this to the developers of the AI to make sure no mistake has been made and an innocent player has suffered the consequences.
> 
> At this moment I will not be banning anyone since no credible and repeatable bugs/exploits have been presented.
> 
> Have a great week,
> 
> * Patricia

INNOCENT?? the scripter deserves his entre steam account banned plus his main plus any other alt he has. you banned a smurf account of moonstruck. did you look at any of the links i sent at all???

**Comment by PatriciaHR-Valve** [comment]
Hello!

Innocent yes. The team is currently looking into the case to make sure no innocent customer has been banned.

I can assure you, no evidence of scripts has been posted in this ticket thus no bans has been made. What is a moonstruck?

I have reviewed the clips you have provided. No signs of foul play has been detected. I am contemplating on charging you with fraud and stealing company time. Please make sure next time you post a clip it has illegal activity in it. Failing to do so will result in a deletion of this ticket from our servers and all further assistance bring revoked.

- Patricia

**Comment by bradleydragon** [comment]
> Hello!
> 
> Innocent yes. The team is currently looking into the case to make sure no innocent customer has been banned.
> 
> I can assure you, no evidence of scripts has been posted in this ticket thus no bans has been made. What is a moonstruck?
> 
> I have reviewed the clips you have provided. No signs of foul play has been detected. I am contemplating on charging you with fraud and stealing company time. Please make sure next time you post a clip it has illegal activity in it. Failing to do so will result in a deletion of this ticket from our servers and all further assistance bring revoked.
> 
> * Patricia

how about you provide proof you are a valve employee 800?

**Comment by actual800** [comment]
> > Hello!
> > Innocent yes. The team is currently looking into the case to make sure no innocent customer has been banned.
> > I can assure you, no evidence of scripts has been posted in this ticket thus no bans has been made. What is a moonstruck?
> > I have reviewed the clips you have provided. No signs of foul play has been detected. I am contemplating on charging you with fraud and stealing company time. Please make sure next time you post a clip it has illegal activity in it. Failing to do so will result in a deletion of this ticket from our servers and all further assistance bring revoked.
> > 
> > * Patricia
> 
> how about you provide proof you are a valve employee 800?

i wouldnt hide behind a fake account to ever speak to you you fat retarded ped9ph1le fa88ot, go jerk off to child p 0rn or be asx tiality or w/e youre into with your swamp hog me th creature before i buy the both of you as livestock and boil you alive

**Comment by Nallefjant** [comment]
lol
